stream_alert/alert_processor/main.py

```diff
@@ -44,11 +47,14 @@ def handler(event, context):
             loaded_sns_message = json.loads(sns_message)
         except ValueError as err:
             LOGGER.error('an error occurred while decoding message to JSON: %s', err)
-            return
+            continue
 
-        if not 'default' in loaded_sns_message:
+        if not 'default' in loaded_sns_message and \
+            'AlarmName' in loaded_sns_message:
+            continue    # skip sns messages related to alarms and do not log them
+        elif not 'default' in loaded_sns_message:
             LOGGER.error('Malformed SNS: %s', loaded_sns_message)
-            return
+            continue
```

Comment:
I've never been a fan of line continuation. In any case, lines 52-57 can be simplified:
```python
if not 'default' in loaded_sns_message:
    if not 'AlarmName' in loaded_sns_message:
        LOGGER.error('...')
    continue
```